art/estimators/certification/deep_z/pytorch.py

```diff
@@ -210,3 +290,82 @@ def certify(self, cent: np.ndarray, eps: np.ndarray, prediction: int) -> bool:
                 certification_results.append(cert_via_sub)
 
         return all(certification_results)
+
+    def concrete_loss(self, output: "torch.Tensor", target: "torch.Tensor") -> "torch.Tensor":
+        """
+        Access function to get the classifier loss
+
+        :param output: model predictions
+        :param target: ground truth labels
+
+        :return: loss value
+        """
+        return self._loss(output, target)
+
+    def apply_preprocessing(self, x: np.ndarray, y: np.ndarray, fit: bool) -> Tuple[Any, Any]:
```

Comment:
Why do we need this method? It seems to just warp the internal method.

Reply:
It was just to avoid a W0212 pylint warning in the certified trainer script. We can remove this and access the wrapped method with a # pylint: disable=W0212 if preferred?